File: rest/taskrouter/workflows/instance/delete/example-1/example-1.rb
# Get twilio-ruby from twilio.com/docs/ruby/install
require 'rubygems'          # This line not needed for ruby > 1.8
require 'twilio-ruby'

# Get your Account Sid and Auth Token from twilio.com/user/account
account_sid = '{{ account_sid }}'
auth_token = '{{ auth_token }}'
workspace_sid = '{{ workspace_sid }}'
activity_sid = '{{ activity_sid }}'

client = Twilio::REST::TaskRouterClient.new account_sid, auth_token, workspace_sid

activity = client.workspace.activities.get(activity_sid)
activity.delete

File: rest/taskrouter/worker-reservations/instance-get-example1/example-1.py
# Download the Python helper library from twilio.com/docs/python/install
from twilio.rest import TwilioTaskRouterClient

# Your Account Sid and Auth Token from twilio.com/user/account
account_sid = "{{ account_sid }}"
auth_token  = "{{ auth_token }}"
workspace_sid = "{{ workspace_sid }}"
task_sid = "{{ task_sid }}"

client = TwilioTaskRouterClient(account_sid, auth_token)

reservation = client.workers(workspace_sid).get(worker_sid).reservations.get(reservation_sid)
print reservation.reservation_status
print reservation.worker_name